documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -67,10 +67,10 @@ Cruise Control uses _capacity limits_ to determine if certain resource-based opt
 You specify capacity limits for Kafka broker resources in the `brokerCapacity` property in `Kafka.spec.cruiseControl` .
 Capacity limits can be set for the following broker resources in the described units:
 
-* `disk`            - Disk storage in bytes
-* `cpuUtilization`  - CPU utilization as a percent (0-100)
-* `inboundNetwork`  - Inbound network throughput in bytes per second
-* `outboundNetwork` - Outbound network throughput in bytes per second
+* `disk`            - Disk storage in megabytes (Default: 100000)
```

Comment:
Is this disk capacity ... of the whole cluster? Or is it per broker?

Reply:
It is per broker.